Greenland attenuation

christophwelling opened the pull request:
Adds new attenuation model based on the measurements taken at Summit last summer:  https://arxiv.org/pdf/2201.07846.pdf

Here is what it looks like: GL1 is the old model, GL2 the new one.

![attenuation_length_comparison](https://user-images.githubusercontent.com/42930879/151555792-29099a56-08d4-413b-90d1-6036c1b18f01.png)

cg-laser commented:
thanks!
why did you force it to return at least 100m? 
Can you also update the changelog?

christophwelling commented:
I updated the changelog.
The 100m cut is also implemented like this in the GL1 model and in the code that Dan sent me (which this is based on).
I assumed that there was a good reason this was put into the GL1 model, but I can remove it.

christophwelling commented:
I talked to Dan about it, and we think that this is most likely to prevent the attenuation length from becoming negative for high frequencies. I reduced the minimum attenuation length to 1m. That will still prevent it from being negative, but only affect values that are already so small that it won't have a practical effect.

cg-laser commented:
sounds good, thanks

christophwelling commented:
So, could you approve this PR? ;-)

christophwelling commented on changelog.txt:
```diff
@@ -19,6 +19,9 @@ secondary interactions. With this bug, the initial interaction needs to be idenf
 This bug does not effect any effective volume calculation. 
 
 version 2.1.5
+new features:
```
Okay, fixed it
